### PR TITLE
fix(Dropdown): aria-hidden elements should not be a focusable element.

### PR DESCRIPTION
**Pull request by imoutaharik**
## Definition of Done
- [ ] User interface is compliant with WCAG 2.1 AA. Ensure these items are fulfilled:
    - [ ] Keyboard operability: All functionality can be accessed using only the keyboard
    - [ ] Logical tab order: Interactive elements follow a consistent and logical tab order
    - [ ] Visual indication of focus: Focused elements are visually highlighted for keyboard navigation

### CODE REVIEW

File: src/components/Trigger/Trigger.tsx [hunk 1]
```diff
@@ -113,14 +113,13 @@ export const Trigger = ({
                         <IconTrashBin size={IconSize.Size12} />
                     </button>
                 )}
-                <button
+                <div
                     {...buttonProps}
```
Comment (imoutaharik) [inline src/components/Trigger/Trigger.tsx:117]: I think we don't even need those buttonProps, do we?

Comment (ryancarville) [reply src/components/Trigger/Trigger.tsx]: Looks like it was added to handle caret positioning but maybe this is outdated now?

**Comment by imoutaharik** [comment]
@ryancarville Yes, caret should not be interactive nor focusable.

**Review by ryancarville — APPROVED** [review]

**Comment by imoutaharik** [comment]
> @ryancarville Yes, caret should not be interactive nor focusable.

But it blocks the input interaction so it's annoying.

**Comment by imoutaharik** [comment]
sync with Lidija. Not needed after all. For now.

**Pull request closed without merging**